bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_NLO_fxfx/Wgamma_Monophoton_Ptg130_NLO_proc_card.dat

```diff
@@ -0,0 +1,20 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model loop_sm-no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define p = p b b~
+define j = j b b~
```

Comment:
Please remove unnecessary define lines that are not being used

Reply:
Removed only define vl = ve vm vt and define vl~ = ve~ vm~ vt~